tomcat9/image.yaml

```diff
@@ -108,20 +113,29 @@ modules:
           - name: dynamic-resources
           - name: s2i-common
           - name: os-jws-s2i
+            version: "3.0"
           - name: jws8-conffiles
             version: '1.0'
           - name: rm-manager
             version: "1.0"
           - name: os-jws-deployments
+            version: "3.0"
           - name: jboss.container.jolokia.bash
           - name: os-jws-jolokia
+            version: "3.0"
           - name: jboss.container.java.jvm.bash
           - name: os-jws-launch
+            version: "3.0"
           - name: os-jws-https
+            version: "3.0"
           - name: jws-secure-mgmt-console
             version: "1.0"
+          - name: os-jws-rm-root
+            version: "3.0"
```

Comment:
Hi @mdrafiur ,  I see you're reintroducing this module, which you had removed with the following commit: https://github.com/jboss-container-images/jboss-webserver-5-openshift-image/commit/72aa523a55b7ed1d417ac4cdc532dc7fbb01c77b#diff-086192c89e99ccbae5d0a32ce0a14a32f22fec77aa4f0604bce83cf524850679 . If you're certain that wasn't done by mistake, it looks good to me.

Reply:
Thanks, @bekionel for identifying the commit. Right, it should be removed. I have removed the 'os-jws-rm-root' module.